### DISCUSSION TITLE
does globe platform support docker container?

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [invertase/globe#16] [Q&A]
Can I deploy a background docker container here?

**Comment by lesnitsky** [comment]
Globe only supports deploying dart apps with an http server or flutter web.
What is your use-case for a background docker container?

**Reply by ClarissaGazalaEvanthe** [reply]
I want to deploy rest api server ai, my DART program requires ffi access

**Reply by lesnitsky** [reply]
Globe currently does not support deploying Dart apps that use FFI.
This is something we will be implementing in the future.
